dev/devicelab/lib/framework/apk_utils.dart

```diff
@@ -83,6 +83,83 @@ bool hasMultipleOccurrences(String text, Pattern pattern) {
   return text.indexOf(pattern) != text.lastIndexOf(pattern);
 }
 
+
+/// Utility class to analyze the content inside an APK.
+class ApkExtractor {
+  ApkExtractor(this.apkFile);
+
+  /// The APK.
+  final File apkFile;
+
+  bool _extracted = false;
+
+  Directory _outputDir;
+
+  Future<void> _extractApk() async {
+    if (_extracted) {
+      return;
+    }
+    _outputDir = await apkFile.parent.createTemp('apk');
+    await eval('unzip', <String>[apkFile.path, '-d', _outputDir.path]);
```

Comment:
This won't work on Windows.  Maybe we don't care about that right now, but it'd be nice to know this test could run on Windows too.

Since this is in the Flutter repo, we download 7za on Windows and that could be used.  It might be good to have an abstraction over this for unzipping files that uses 7za on Widnows and unzip elsewhere.

Reply:
thanks for pointing this out. I updated it, so that it works on Windows now.